src/tools/vfs/vfs_main.c

```diff
@@ -191,10 +191,15 @@ int vfs_mount(int pid)
     }
 
     ret = mkdir(mountpoint, S_IRWXU);
-    if ((ret < 0) && (errno != EEXIST)) {
-        ret = -errno;
-        vfs_error("failed to create directory '%s': %m", mountpoint);
-        goto out_free_mountpoint;
+    if (ret < 0) {
```

Comment:
I don't mind the change, but in this place there is no issue even without this fix, because `ret` gets assigned a new value on line 207. But up to you

Reply:
If we ignore `EEXIST` than IMO ret should be 0 because it reflects the status of this code section